server/S3.js

```diff
@@ -0,0 +1,42 @@
+require('dotenv').config();
+const fs = require('fs')
+const S3 = require('aws-sdk/clients/s3');
+
+const bucketName = process.env.AWS_S3_BUCKET_NAME;
+const region = process.env.AWS_S3_BUCKET_REGION;
+const accessKeyId = process.env.AWS_ACCESS_KEY;
+const secretAccessKey = process.env.AWS_SECRET_ACCESS_KEY;
+
+const s3 = new S3({
+  region,
+  accessKeyId,
+  secretAccessKey,
+})
+
+// Function to handle uploading to S3 bucket
+function uploadPhoto(file) {
```

Comment:
Decorate with `async` keyword since it returns a `Promise`

Reply:
Where the function is call - it is async.